tests/e2e/e2e_groups_test.go

```diff
@@ -1,14 +1,14 @@
 package e2e
```

Comment:
Missing some docs to explain the flows that are being tested

Reply:
I'm not sure why we would have docs for a test, the acceptance criteria/tested flows were part of the original ticket for this. There's sufficient logging in this suite to understand what is being tested.

Reply:
There's also a qa document in hub-eng that explains group flows in better detail

Reply:
An overview of whats being tested without having a deep dive into the code/read through the logs helps. Remember this is an open source project, other contributors don't have access to hub-eng.

Reply:
Totally fair, let's open an issue then to make sure it's consistent across test suites